bolt-sidecar/src/config/chain.rs

```diff
@@ -97,6 +97,16 @@ pub enum Chain {
 }
 
 impl Chain {
+    /// Get the chain ID for the given chain.
+    pub const fn id(&self) -> u64 {
```

Comment:
we already have `ChainConfig::chain_id()`. if you want to keep this, please update that function to just call `chain.id()` instead of matching again. I don't mind either one option